rocAL/rocAL/source/api/rocal_api_augmentation.cpp

```diff
@@ -70,6 +70,45 @@ THE SOFTWARE.
 
 #define MAX_ASPECT_RATIO 3.0f
 
+void get_rocal_tensor_layout(RocalTensorLayout &tensor_layout, RocalTensorlayout &op_tensor_layout)
+{
+    switch(tensor_layout)
+    {
+        case 0:
+            op_tensor_layout = RocalTensorlayout::NHWC;
+            return;
+        case 1:
+            op_tensor_layout = RocalTensorlayout::NCHW;
+            return;
+        case 2:
+            op_tensor_layout = RocalTensorlayout::NFHWC;
+            return;
+        case 3:
+            op_tensor_layout = RocalTensorlayout::NFCHW;
+            return;
+        default:
+            THROW("Unsupported Tensor layout" + TOSTR(tensor_layout))
+    }
+}
+
+void get_rocal_tensor_data_type(RocalTensorOutputType &tensor_output_type, RocalTensorDataType &tensor_data_type)
```

Comment:
same here. Why do we need this function since it is just doing a copy

Reply:
This has been removed in recent commits